cmd/image.go

```diff
@@ -66,11 +66,15 @@ func checkImage(container apiv1.Container, image string, result *Result) {
 
 	contTag = contImgLabel[1]
 
+	result.img_name = contImg
+	result.img_tag = contTag
+
 	if contImg == compImg && contTag != compTag {
 		result.err = 2
-		result.img_name = contImg
-		result.img_tag = contTag
 	}
+
+	result.err = 0
```

Comment:
I know this is redundant since the zero value for an `int` is `0`. But I think it is better style to be explicit.